docs/stdlib/numbers.rst

```diff
@@ -355,7 +373,7 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: unary minus subtract
 
-    Arithmetic negation.
+    Performs a logical negation of an arthimetic, arbitrarily numerical value:
```

Comment:
@vpetrovykh Can you speak to the accuracy of this one? It feels off, but I'm not sure.